src/test/java/com/alibaba/wasp/jdbc/TestJdbcStatement.java

```diff
@@ -61,6 +62,7 @@ public static void beforeClass() throws Exception {
 
   @AfterClass
   public static void afterClass() throws Exception {
+    Assume.assumeNotNull(conn);
```

Comment:
Should you skip the test result if you cannot close the connection?

Reply:
I think I have made a mistake here. I should have maybe preserved the if-else check here so that the conn.close() line isn't executed if conn is null. 'Assume' isn't probably the right thing to use here.